src/components/Typography/Typography.stories.mdx

```diff
@@ -369,6 +369,7 @@ export const System = () => {
                         >
                             {config.items.map((item, ind) => (
                                 <div
+                                    key={ind}
```

Comment:
Hey would you like to add more meaningful key instead of the index of array itself? like the one below? 
PS: if you can remove the `* 1` it would be great as well
https://github.com/CRED-CLUB/neopop-web/blob/9e2ffb2052dc8dd51707a836dae662b998a84688/src/components/Typography/Typography.stories.mdx#L351

Reply:
Sure